src/poc/miner_poc_statem.erl

```diff
@@ -110,10 +110,21 @@ init(Args) ->
         {error, _} ->
             {ok, requesting, #data{base_dir=BaseDir, blockchain=Blockchain,
                                    address=Address, poc_interval=Delay, state=requesting}};
+        %% support for some states were removed but some miners may have it saved in their state file
+        %% so we will force these to default to requesting
+        {ok, _State, _Data} when _State =:= targeting;
```

Comment:
Would this not be brittle if we change it again? Any way to just catch the states we support instead?

Reply:
good point, will do that

Reply:
@madninja now only supports exported states